airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DbConverter.java

```diff
@@ -32,15 +35,14 @@
 import io.airbyte.config.WorkspaceServiceAccount;
 import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
 import io.airbyte.protocol.models.ConnectorSpecification;
-import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 import org.jooq.Record;
 
 public class DbConverter {
```

Comment:
Sorry I know this was written before this changeset - do you mind dropping a javadocs here so it's clear what this class is for for future readers?

Reply:
Sure!